stackslib/src/core/mod.rs

```diff
@@ -68,9 +68,9 @@ pub const GENESIS_EPOCH: StacksEpochId = StacksEpochId::Epoch20;
 /// The number of blocks which will share the block bonus
 ///   from burn blocks that occurred without a sortition.
 ///   (See: https://forum.stacks.org/t/pox-consensus-and-stx-future-supply)
-#[cfg(test)]
+#[cfg(any(test, feature = "testing"))]
 pub const INITIAL_MINING_BONUS_WINDOW: u16 = 10;
-#[cfg(not(test))]
+#[cfg(not(any(test, feature = "testing")))]
 pub const INITIAL_MINING_BONUS_WINDOW: u16 = 10_000;
```

Comment:
This is pretty scary to me -- the `testing` feature is enabled by default for `debug` builds in the `stacks-node`, meaning this constant would change depending on whether or not you built `stacks-node` as `--release`.